mex/ifsg/transform.py

```diff
@@ -452,10 +447,9 @@ def transform_ifsg_data_to_mex_variables(
         in [c2i2s.id_catalogue2item for c2i2s in meta_catalogue2item2schema]
     ]
     for row in filtered_variables:
-        belongs_to = []
-        if row.statement_area_group in label_by_statement_area_group:
-            label = label_by_statement_area_group[row.statement_area_group]
-            belongs_to.append(variable_group_stable_target_id_by_label[label])
+        belongs_to = variable_group_by_identifier_in_primary_source.get(
+            f"{row.id_type}_{row.statement_area_group}"
+        )
```

Comment:
`.get` returns `None` if the key is not found. Based on the mapping I would assume that we always want to set `belongs_to` and hence this should fail if the key is not in `variable_group_by_identifier_in_primary_source`.

```suggestion
        belongs_to = variable_group_by_identifier_in_primary_source[
            f"{row.id_type}_{row.statement_area_group}"
        ]
```

Reply:
the mapping README states
```
For each disease (identified by using the *Type* attribute `IdType`), we want to form a distinct variable group, meaning that e.g. the group "Epi" will be applicable for each disease. This allows us to specify by individual diseases for each variable group and not to map all existing variables of all diseases in one variable group.
```
Not all variables correspond to diseases and in conclusion not all variables can be assigned to variable-groups. 
And indeed your suggested changes fail, because `statement_area_group` can be `None` as the `id_type` doesn't correspond to disease.